src/config.rb

```diff
@@ -63,11 +67,26 @@ def initialize(file = nil, options = {})
       end
     end
 
+    def loader
+      @loader ||= Validator::Loader.new(self)
+    end
+
+    def saver
+      @saver ||= Validator::Saver.new(self)
+    end
```

Comment:
Ok, I think at this point I only use the file loader built into the `BaseCommand` so it can be removed from the `Config`